sdk/react/panels/StartScreen.js

```diff
@@ -10,20 +9,20 @@ import {
   TouchableHighlight,
 } from 'react-native';
 
-var Utils = require('../utils');
-var Log = require('../log');
-var styles = Utils.getStyles(require('./style/startScreenStyles.json'));
-var Constants = require('../constants');
-var {
+const Utils = require('../utils');
+const Log = require('../log');
+const styles = Utils.getStyles(require('./style/startScreenStyles.json'));
```

Comment:
I know it was there before but is there a reason to not start this var name with capital S?  All the other ones are capital. Or should we make all the other lowercase? @aeng7 @dallen42  What do you guys think??

Reply:
@pilievOoyala because style in this case is object, and other variables is Components with functions